standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/conf/MetastoreConf.java

```diff
@@ -701,6 +701,9 @@ public enum ConfVars {
         "datanucleus.connectionPoolingType", "HikariCP", new StringSetValidator("DBCP",
         "HikariCP", "NONE"),
         "Specify connection pool library for datanucleus"),
+    CONNECTION_POOLING_LIFETIME("datanucleus.connectionLifetime",
+        "datanucleus.connectionLifetime", 1800,
+        "The maximum lifetime, in seconds, of a connection in the pool, used in HikariCP only."),
```

Comment:
If it is a Hikari only configuration then it shouldn't be part of `MetastoreConf`. Also it is strange to have this under `datanucleus` namespace if it is not a datanucleus configuration.